modular_skyrat/modules/emotes/code/silicon_emotes.dm

```diff
@@ -1,4 +1,4 @@
-/datum/emote/silicon
+/* /datum/emote/silicon
```

Comment:
Reeeemove the file if it's being deleted ;-; I need to make a new contrib update don't I?